shared/js/dataDriver.js

```diff
@@ -277,13 +292,15 @@ loop.DataDriver = function() {
      * @param {String} token The room token to use for connections.
      */
     _connectToRoom(token) {
+      this._retryTimeout = this.INITIAL_RETRY_TIMEOUT;
       this._roomToken = token;
       this._sse = new EventSource(this._buildUrl() + "?" + this._buildQuery({
         limitToLast: 1,
         orderBy: '"timestamp"'
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
Yes. I tested by connecting, disconnecting my ethernet cable, and observing the error handler being called (with log messages). It then tries to reconnect, but can't, and the error handler is called again. The exponential backoff is operational.